src/runtime/api-channel.ts

```diff
@@ -442,6 +456,7 @@ export abstract class PECOuterPort extends APIPort {
   @NoArgs Stop() {}
   DefineHandle(@RedundantInitializer store: StorageProviderBase, @ByLiteral(Type) type: Type, @Direct name: string) {}
   InstantiateParticle(@Initializer particle: recipeParticle.Particle, @Identifier @Direct id: string, @ByLiteral(ParticleSpec) spec: ParticleSpec, @ObjectMap(MappingType.Direct, MappingType.Mapped) stores: Map<string, StorageProviderBase>) {}
+  RebootParticle(@RebootInitializer particle: recipeParticle.Particle, @Identifier @Direct id: string) {}
```

Comment:
Is RebootParticle a good name? I know I used it in my hacky patch - but in that patch I didn't care about names much, but names are important. This is the moment when it's easiest to change this name, it will never be as easy as it is today. Whatever we decide, other engineers for years to come will likely need to live with.

So I'm wondering... Would `ReloadParticle` work better? It has the connotation to "Hot Code Reload" / "Hot Reload" which are phrases known among engineers. "Reboot" has no such connotations, so it will introduce more mental friction when reading the code and explaining to people.

WDYT? (What do you think - I use this acronym a lot :) )

Reply:
Sounds good, changed everything to using reload instead of reboot :D